### PR TITLE
Create Education Benefits Wizard on 1990

### PR DESCRIPTION
**Pull request by maharielrosario**
## Description
This PR sets up an education benefits wizard on the 1990 introduction page.

## Testing done

- Unit tests have been created

## Screenshots
screenshot-localhost_3001-2020 07 27-15_47_25
screenshot-localhost_3001-2020 07 27-15_48_23

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/edu-benefits/1990/containers/IntroductionPage.jsx [hunk 1]
```diff
@@ -3,114 +3,688 @@ import { focusElement } from 'platform/utilities/ui';
...
+    this.state = []
+      .concat(...levels)
+      .reduce((state, field) => Object.assign(state, { [field]: null }), {
+        open: false,
+        educationBenefitSelected:
+          sessionStorage.getItem('educationBenefitSelected') || 'none selected',
+        wizardCompletionStatus:
+          sessionStorage.getItem('EduWizardStatus') || 'not complete',
+      });
+  }
```
Comment (bradl3yC) [inline src/applications/edu-benefits/1990/containers/IntroductionPage.jsx:34]: Can we get rid of this chained expression implementation of state and just use this:

```
state = {
  open: false,
  educationBenefitSelected: sessionStorage.getItem('educationBenefitSelected') || 'none selected',,
  wizardCompletionStatus: sessionStorage.getItem('EduWizardStatus') || 'not complete',
  newBenefit: null,
  serviceBenefitBasedOn: null,
  transferredEduBenefits: null,
  nationalCallToService: null,
  sponsorDeceasedDisabledMIA: null,
  vetTecBenefit: null,
  sponsorTransferredBenefits: null,
  applyForScholarship: null
}
```

File: src/applications/edu-benefits/1990/containers/IntroductionPage.jsx [hunk 2]
```diff
@@ -3,114 +3,688 @@ import { focusElement } from 'platform/utilities/ui';
...
+    const form1995 = showSTEMScholarship ? '10203' : '1995';
+    if (wizardCompletionStatus === 'pending') {
+      if (
+        newBenefit === 'yes' &&
+        nationalCallToService === 'no' &&
+        vetTecBenefit === 'no'
+      ) {
+        this.setEduBenefitFormSelected('1990');
+        this.setWizardCompletionStatus('complete');
+      } else if (
```
Comment (bradl3yC) [inline src/applications/edu-benefits/1990/containers/IntroductionPage.jsx:61]: can we remove the use of else and just let the if statements remain on their own line (guard clauses)

File: src/applications/edu-benefits/1990/containers/IntroductionPage.jsx (hunk 1 above)
Comment (mr0sari0) [reply src/applications/edu-benefits/1990/containers/IntroductionPage.jsx]: Got it!

**Comment by mr0sari0** [comment]
Replaced by https://github.com/department-of-veterans-affairs/vets-website/pull/13632

**Pull request closed without merging**